src/components/utils/Header1.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+    <h1 id="page-title" tabindex="-1">
+      <span v-if="text">{{ text }}</span>
+      <span v-if="!text"><slot /></span>
+    </h1>
+  </template>
+  
+  <script>
+  export default {
+    name: 'Header1',
+    props: {
+      text: {
+        default: undefined,
+        required: false,
+        type: String
+      }
+    }
+  }
+  </script>
```

Comment:
Indentation fix needed above. (Would the front-end linter catch this?)

Reply:
Would love for the linter to fix this for me! I guess the way I used it, it did not catch this? https://github.com/gmerritt/hartsfield-from-ripley/commit/a44aa2ade61bde3158b2e27eafa2ce29d98fc56f

Reply:
@gmerritt - Maybe this eslint will make the check and fix?! https://eslint.vuejs.org/rules/script-indent